webtool/test_api.py

```
@@ -1821,6 +1825,133 @@ def test_erfundener_merker_schiebt_keine_gesunde_datei_beiseite(client, tmp_path
+    Genau die Luecke aus #239.
+
+    Beide Wege muessen liefern: die gebaute Fassung (noch keine edit.json) und die gelesene."""
+    ohne = client.get("/api/projects/Demo/files/S1").json()
+    assert ohne["dateistand"] == "", "ohne edit.json ist der Stand die leere Zeichenkette"
+    client.put("/api/projects/Demo/files/S1", json=ohne)
+    mit = client.get("/api/projects/Demo/files/S1").json()
+    assert mit["dateistand"], "mit edit.json muss ein Stand geliefert werden"
+    assert (tmp_path / "Demo" / "transkripte" / "S1.edit.json").exists()
+
+
+def test_zwei_schreibvorgaenge_gleicher_groesse_haben_verschiedene_staende(client, tmp_path):
```

Ruff (0.16.1)

[warning] Unused function argument: `client`

(ARG001)